app/grandchallenge/evaluation/templates/evaluation/evaluation_detail.html

```diff
@@ -247,20 +254,41 @@ <h3 class="card-title">Prerequisite Jobs</h3>
                 </div>
             {% endif %}
 
+            <div class="card-body">
+                <h3 class="card-title">Results</h3>
```

Comment:
Results says to me here "The results of this evaluation". Would Predictions be a better title?

Reply:
Good point. I had some confusion about this as well. Looking at the different subsections I'm also spotting a predictions.json download button under 'Predictions'. Going to combine them.